src/parsing/parser_types.rs

```diff
@@ -17,13 +21,20 @@ impl<'a> Context<'a> {
         configuration: &'a Configuration,
         format_code_block_text: Box<dyn Fn(&str, &str, u32) -> Result<String, String>>
     ) -> Context<'a> {
+        let ignore_regex = format!(r"\s*<!\-\-\s*{}\s*\-\->\s*", configuration.ignore_directive);
+        let ignore_start_regex = format!(r"\s*<!\-\-\s*{}\s*\-\->\s*", configuration.ignore_start_directive);
+        let ignore_end_regex = format!(r"\s*<!\-\-\s*{}\s*\-\->\s*", configuration.ignore_end_directive);
```

Comment:
I wonder if there's an easy way to escape the text being provided for the regex.